src/main/kotlin/io/openfuture/state/blockchain/dto/UnifiedTransaction.kt

```diff
@@ -4,7 +4,7 @@ import java.math.BigDecimal
 
```

Comment:
I don't think that it a good idea to use the same model for different structures of data. What if next chain would require maps?

Reply:
to some extent I agree with you, but overall structure of blockchains are same

Reply:
i think we should discuss